.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Declare files that will always have CRLF line endings on checkout.
+*.sh text eol=lf
+*.py text eol=lf
+Makefile text eol=lf
```

Comment:
I'm confused.

Is this for Windows or something?  Why not just check them in with the correct formatting?